packages/admin/cms-admin/src/generator/future/generator.ts

```diff
@@ -56,6 +56,10 @@ export type GridConfig<T extends { __typename?: string }> = {
     delete?: boolean;
     copyPaste?: boolean;
     readOnly?: boolean;
+    actions?: {
+        columnWidth?: number;
+        componentImport?: ImportReference;
```

Comment:
the ImportReference for block/validate is not named with ...Import, I would also not do that here

Reply:
> Can the import really be optional?

It's possible that someone would like to adjust the column-width independent of wanting to add an additional component. 
Although, when there is no additional component the width should probably be calculated, depending on how many actions are shown (Edit, Context-Menu, nothing) that is currently not the case however.